README.md

```diff
@@ -1004,6 +1004,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Justine Syde Bulacito](https://github.com/celesica)
   - [Jyoti KM](https://github.com/JyotiKM29)
   - [Jyotiraditya](https://github.com/Jyotiraditya24)
+  - [Joel Aliyu](https://github.com/joelali5)
```

Comment:
Please add your name in line 974, after Joachim Chisom.
Let us know if you need any help with that. 🙂

Reply:
I have made the changes